src/docs/asciidoc/Application-Configuration.asciidoc

```diff
@@ -310,52 +310,112 @@ will be attempted repeatedly, with no termination. Also see `republishToDLQ`
 If you want to use Kafka as a data transport, the following connection settings, as well as defaults for the kafka
 message bus can be modified in `servers.yml`.
 
-NOTE: To ensure the proper functioning of the Kafka Message Bus, you must eanble log cleaning in your Kafka configuration.  This is set using the configuration variable `log.cleaner.enable=true`.  See the https://cwiki.apache.org/confluence/display/KAFKA/Log+Compaction[Kafka documentation] for additional configuration options for log cleaning.
+NOTE: To ensure the proper functioning of the Kafka Message Bus, you must eanble log cleaning in your Kafka
+configuration.  This is set using the configuration variable `log.cleaner.enable=true`.
+See the https://cwiki.apache.org/confluence/display/KAFKA/Log+Compaction[Kafka documentation] for additional
+configuration options for log cleaning.
 
-NOTE: At this time, the Kafka message bus does not support job processing. This feature will be available in a future release.
+NOTE: At this time, the Kafka message bus does not support job processing.
+
+NOTE: The Kafka message bus does not support `count=0` for module deployments, and therefore, it does not support
+direct binding of modules. This feature will be available in a future release. In the mean time, if direct communication
+ between modules is necessary for Kafka deployments, xref:Modules#composing-modules[composite modules] should be used instead.
 
 [source,yaml]
 ----
   messagebus:
     kafka:
```

Comment:
Correct - the former is a slip, the second is not available as an option anymore either (0.8.2 producers are always batching) - will remove.